### PR TITLE
[FIX] *: retarget links and references to 15.0

### PR DESCRIPTION
**Pull request by Feyensv**
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: odoo/http.py [hunk 1]
```diff
@@ -785,7 +785,7 @@ def dispatch(self):
 
 Odoo URLs are CSRF-protected by default (when accessed with unsafe
 HTTP methods). See
-https://www.odoo.com/documentation/14.0/developer/reference/http.html#csrf for
+https://www.odoo.com/documentation/15.0/developer/reference/http.html for
```
Comment (mart-e) [inline odoo/http.py:788]: https://www.odoo.com/documentation/15.0/developer/reference/addons/http.html#csrf

**Review by mart-e — COMMENTED** [review]
I think `addons/l10n_cl/__manifest__.py` needs to be updated too

**Comment by mart-e** [comment]
@robodoo r+ rebase-ff

**Comment by sswapnesh** [comment]
@Feyensv 
Screenshot from 2021-10-05 23-43-05
Seems like these are missed

**Pull request closed without merging**